docs/output-data/core-cne/java-api.md

```diff
@@ -1,41 +1,39 @@
-After completing the RAO, the user can export the CORE CNE file using this method of [CoreCneExporter](https://github.com/powsybl/powsybl-open-rao/blob/main/data/result-exporter/core-cne-exporter/src/main/java/com/powsybl/openrao/data/corecneexporter/CoreCneExporter.java):
+After completing the RAO, the user can export the [`RaoResult`](/output-data/rao-result.md) object as a CORE CNE file using the `write` method with the `"CORE-CNE""` format:
 
 ~~~java
-public void exportCne(Crac crac,
-    UcteCracCreationContext cracCreationContext,
-    RaoResult raoResult, RaoParameters raoParameters,
-    CneExporterParameters exporterParameters, OutputStream outputStream)
+// RaoResult interface
+public void write(CracCreationContext cracCreationContext, Properties properties, OutputStream outputStream)
```

Comment:
you should put here an example the user can practically use. so put the method with the format argument